src/main/scala/scalaz/nio/Buffer.scala

```diff
@@ -18,30 +18,30 @@ import scala.reflect.ClassTag
 abstract class Buffer[A: ClassTag] private[nio] (private[nio] val buffer: JBuffer) {
   final val capacity: UIO[Int] = ZIO.succeed(buffer.capacity)
 
-  final def position: UIO[Int] = IO.effectTotal(buffer.position)
+  final val position: UIO[Int] = IO.effectTotal(buffer.position)
```

Comment:
i'm not sure if we can do `val` here since buffer is mutable and `position` as well as`limit`, `remaining`... will potentially change.